src/util.cc

```diff
@@ -56,7 +56,7 @@
 
 void Fatal(const char* msg, ...) {
   va_list ap;
-  fprintf(stderr, "ninja: fatal: ");
+  fprintf(stderr, ": ninja: fatal: ");
```

Comment:
When a test fails, the ninja was merged with the test name making it more difficult than necessary to copy/paste it in the exclusion list. At the minimum there should be a space or a \n before "ninja:".

Reply:
`Fatal` is also used by non-test code though.

Reply:
But leaving a space after « : » is standard, no?

Reply:
Yes, but previously there wasn't even a `:`.

Reply:
This is not part of the main feature here, so if you do not want it, I can remove it.

That being said, the issue here is that this blindly get appended to the error message (for instance when the ninja test fails, “ninja” ends up mangled at the end of the failing test name making it impossible to efficiently select (double click on the failing test name also copies ninja). Ideally this message would make sure it start with a separator(new line or :, ...) or we need to enforce that any error will end with a separator... it seems simpler to do it centrally and a new line seems overkill, so I opted for the compact “: “. If the message ends up with a new line, the extra “: “ at the beginning of the line seems less of an issue than the problem it fixes.

Reply:
I see. Might be something fixed by using GoogleTest (see #1562).

> This is not part of the main feature here, so if you do not want it, I can remove it from this pull request.

Yes, please do.